indico/cli/user.py

```diff
@@ -166,3 +166,39 @@ def revoke_admin(user_id):
         user.is_admin = False
         db.session.commit()
         print(cformat("%{green}Administration rights revoked successfully"))
+
+
+@cli.command()
+@click.argument('user_id', type=int)
+def block(user_id):
+    """Blocks a given user"""
+    user = User.get(user_id)
+    if user is None:
+        print(cformat("%{red}This user does not exist"))
+        return
+    _print_user_info(user)
```

Comment:
I'd print an extra line in that function if the user is blocked (and nothing if they aren't, since blocked users are somewhat uncommon)